docs/user/datastores/index_config.rst

```diff
@@ -638,6 +638,9 @@ Note that this configuration will prevent missing visibility labels in normal wr
 it is still possible to write unlabelled data through older clients, bulk loading, or direct access
 to the underlying database.
 
+In Accumulo data stores, setting this configuration will also set the Accumulo ``ReqVisFilter`` on all data
+tables, which will prevent any unlabelled data from being returned in queries.
```

Comment:
Since we are setting the ReqVisFilter on scan, majc, and minc, this will do more than prevent data from being returned in queries.  It ought to be prevent data without vises from being persisted in the database or remaining in the database (if the configuration is added).

Part of me wonders if we should separate the GeoMesa writing concern from the stronger guarantees in Accumulo.

Reply:
it will only delete things when they get compacted though, which potentially might never happen?

Reply:
Hehehe, we are into corner cases on corner cases.  Generally, I think it is safe to assume that all tables are going to see minor and major compactions.  

From https://accumulo.apache.org/docs/2.x/configuration/server-properties there are two table.compaction.*.idle properties that say that when a tablet is idle for a period of time a compaction may happen.  (For minc, there is a note that says "There is no guarantee an idle tablet will be compacted."....)

I suppose I'm trying to figure out if there'd a situation where someone would want to have the write protection but not have the Accumulo iterators configured?  This one is obviously tough:).

Reply:
i can't really think of a situation where someone would want one and not the other... if they really wanted to they could go remove the iterator configs by hand.